setup.cfg

```diff
@@ -1,25 +0,0 @@
-[metadata]
-license_files = LICENSE.txt
-
-[tool:autover]
-reponame = nbsite
-
-[flake8]
-# F - all pyflake errors
-# E101 - indentation contains mixed spaces and tabs
-# E111 - indentation is not a multiple of four
-# E501 - line too long (see max-line-length)
-# E722 - do not use bare except, specify exception instead
-include = *.py
-exclude = .git,__pycache__,.tox,.eggs,*.egg,doc,dist,build,_build,examples,.ipynb_checkpoints,node_modules,apps
-ignore = E,W,F812
-select = F,E101,E111,E501,E722
-max-line-length = 165
-per-file-ignores = panel/tests/ui/jupyter_server_test_config.py:F821
-
-[isort]
-force_grid_wrap=4
-multi_line_output=5
-combine_as_imports=True
-lines_between_types=1
-include_trailing_comma=True
```

Comment:
Add this to pyproject toml as `[tool.isort]`. This will also revert all the import changes.